packages/salesforcedx-sobjects-faux-generator/test/integration/integrationTestUtil.ts

```diff
@@ -11,9 +11,25 @@ import {
   SfdxCommandBuilder
 } from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
 import childProcess = require('child_process');
+import * as fs from 'fs';
 import * as path from 'path';
 import * as util from 'util';
 
+// Used only for CI purposes. Must call delete if you call create
+export function createCIKey(keyLocation: string) {
+  const SERVER_KEY = process.env.SFDX_CI_DEVHUB_JWTKEY;
```

Comment:
For local development/testing, this environment variable is not necessary as long as you have a default devhub set.